plugins/ssh-plugin/src/ssh-plugin-backend.ts

```diff
@@ -124,14 +132,18 @@ const createKeyPair = async (sshkeyManager: SshKeyManager) => {
         await updateConfig(hostName);
         await writeKey(hostName, privateKey);
         await theia.window.showInformationMessage(`Key pair for ${hostName} successfully created`);
-        showWarning();
+        showReadyToUse();
     } catch (error) {
         theia.window.showErrorMessage(error);
     }
 };
 
 const deleteKeyPair = async (sshkeyManager: SshKeyManager) => {
     const keys: cheApi.ssh.SshPair[] = await sshkeyManager.getAll('vcs');
+    if (keys.length < 1) {
```

Comment:
The array check can be moved inside a function, it is used more than in one place